oidc-service-impl/src/test/java/io/mosip/esignet/services/OAuthServiceTest.java

```diff
@@ -425,7 +425,7 @@ public void getTokens_withInvalidRedirectUri_thenFail() {
         OIDCTransaction oidcTransaction = new OIDCTransaction();
         oidcTransaction.setKycToken("kyc-token");
         oidcTransaction.setClientId("client-id");
-        oidcTransaction.setRedirectUri("https://test-redirect-uri/test/test-page");
+        oidcTransaction.setRedirectUri("https://test-redirect-uri1/**");
```

Comment:
In  OIDCTransaction we will never have a pattern, it will always be an absolute URL